### PR TITLE
add new fixed param for collision safety margin

### PR DESCRIPTION
**Pull request by MedericFourmy**

### CODE REVIEW

File: src/crocoddyl_reaching.cpp [hunk 1]
```diff
@@ -71,11 +71,9 @@ namespace panda_torque_mpc
...
         auto runningConstraintModelManager =  boost::make_shared<crocoddyl::ConstraintModelManager>(state);
         auto terminalConstraintModelManager =  boost::make_shared<crocoddyl::ConstraintModelManager>(state);
-        Eigen::VectorXd lower_bound(1);
-        Eigen::VectorXd upper_bound(1);
-
-        lower_bound << 5e-2;
-        upper_bound << std::numeric_limits<double>::infinity();
+        Eigen::VectorXd lb_col(1), ub_col(1);
+        lb_col << config.collision_safety_margin;
+        ub_col << std::numeric_limits<double>::infinity();
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:76]:
```suggestion
        Eigen::VectorXd col_lower_bound(1);
        Eigen::VectorXd col_upper_bound(1);
        col_lower_bound << config.collision_safety_margin;
        col_upper_bound << std::numeric_limits<double>::infinity();
```
Personally, I like this combination more. `lb` and `up` are a bit ambiguous IMO

File: config/controller_configs.yaml [hunk 2]
```diff
@@ -218,6 +218,9 @@ ctrl_mpc_linearized:
     w_u_reg_running: 0.001
     diag_u_reg_running: [1.,1.,1.,1.,1.,1.,1.]
 
+    # Obstacle avoidance
+    collision_safety_margin: 0.20  # in meter
```
Comment (Kotochleb) [inline config/controller_configs.yaml:222]:
```suggestion
    collision_safety_margin: 0.05  # in meter 
```
This change should not change behaviour of the system as a whole

File: src/crocoddyl_reaching.cpp [hunk 3]
```diff
@@ -85,8 +83,8 @@ namespace panda_torque_mpc
             auto constraint = boost::make_shared<crocoddyl::ConstraintModelResidual>(
                 state,
                 obstacle_distance_residual,
-                lower_bound,
-                upper_bound
+                lb_col,
+                ub_col
```
Comment (Kotochleb) [inline src/crocoddyl_reaching.cpp:87]:
```suggestion
                col_lower_bound,
                col_upper_bound
```

**Review by Kotochleb — CHANGES_REQUESTED** [review]

File: config/controller_configs.yaml (hunk 2 above)
Comment (MedericFourmy) [reply config/controller_configs.yaml:222]: will fix

File: config/pose_publisher_params.yaml
Comment (Kotochleb) [inline config/pose_publisher_params.yaml:1]: Unrelated changes

**Review by Kotochleb — CHANGES_REQUESTED** [review]

**Comment by MaximilienNaveau** [comment]
LGTM

**Review by Kotochleb — APPROVED** [review]
Looks good and I after testing I can confirm it works

**Pull request merged**